crates/hsdev/src/tests/mod.rs

```diff
@@ -0,0 +1,44 @@
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use toml::Value;
+
+    use crate::{get_toml_table, input_file::InputData};
+
+    #[test]
+    fn test_input_file() {
+        let toml_str = "username = \"db_user\"\n\
+            password = \"db_pass\"\n\
+            dbname = \"db_name\"\n\
+            host = \"localhost\"\n\
+            port = 5432";
```

Comment:
Optional change: you can take advantage of Rust's raw string literals instead of having to escape quotes:

```rust
let toml_str = r#"username = "db_user"
password = "db_pass"
dbname = "db_name"
host = "localhost"
port = 5432"#;
```